internal/layers/layers.go

```diff
@@ -523,7 +524,7 @@ func getScratchVHDPath(layerFolders []string) (string, error) {
 	// When not sharing a scratch space, `hostPath` will be the path to the sandbox.vhdx to use.
 	//
 	// Evaluate the symlink here (if there is one).
-	hostPath, err := filepath.EvalSymlinks(hostPath)
+	hostPath, err := fs.ResolvePath(hostPath)
 	if err != nil {
 		return "", errors.Wrap(err, "failed to eval symlinks")
```

Comment:
same comment about failure string "failed to normalize path" or "failed to resolve path"?